k8s/yaml/components.yaml

```diff
@@ -633,13 +713,39 @@ spec:
       containers:
       - env:
         - name: SELDON_KAFKA_BOOTSTRAP_SERVERS
-          value: seldon-kafka-bootstrap.seldon-mesh:9092
+          value: 'seldon-kafka-bootstrap.seldon-mesh:9092'
```

Comment:
:information_source: This is a bit inconsistent.  Lots of things are put in the `default` namespace, but Kafka is expected in `seldon-mesh`

Reply:
With the installation discussed in the PR body, this still works fine, although it may not when TLS certs are in use due to the different namespaces?  We could improve that in a follow-up PR